Improve language-switch, add some JavaScript, add colors and such.

domhabersack opened the pull request:
The language-switch is now a dropdown-menu, because why not. Added some JavaScript to make that happen.

Check out the Atomic Design, y’all: the snippet for the language-switcher uses another snippet that only does the styling of the language with the flag.

Commit:
Improve language-switch, add some JavaScript, add colors and such.

Commit:
Improve language-switch, add some JavaScript, add colors and such.

domhabersack commented on .jshintrc:
```diff
@@ -0,0 +1,3 @@
+{
+  "eqeqeq": true
```
Can someone add some stuff here?

stehefan replied:
Why .jshint and not .eslint?

domhabersack replied:
Because this is not cool yet. Do you know how to make it cool?

stehefan replied:
Of course: I'm using it - instacool. And you can fine-tune rules and say what's a warning and what a real error and stuff like this. I think I'd favor it, because I'm using it, but in general jshint is fine :)

domhabersack commented on Gulpfile.js:
```diff
@@ -15,9 +17,20 @@ gulp.task('css', function () {
 
 gulp.task('watch', function () {
+  gulp.watch('assets/js/**/*.js', ['js']).on('change', function () {
+    browserSync.reload();
```
This doesn’t really reload. Any idea as to why that is?

stehefan replied:
I'm not sure if you correctly initialized BrowserSync if I look at their [documentation](https://www.browsersync.io/docs/gulp).

stehefan commented:
Why do we check in compiled assets? Is this really necessary? And flex-wise: we might want to think about using autoprefixer or something similar, otherwise we're cutting out a couple of browsers, don't we? [Caniuse for reference](http://caniuse.com/#feat=flexbox)

domhabersack commented:
Adding Autoprefixer is on the Trello-board.

domhabersack commented:
Also for the compiled assets, because I don’t know how to get Gulp to do its thing on the webhost. No CI, you see.

stehefan commented:
Ah okay, wasn't there on friday. If we don't do our biketour today, I can add it.

Commit:
Add unattractive but functional contact-form.

This sends a reply-ready email to the address that is specified as the
“contact form recipient” in the site-settings.

I would have done something with the colors, but all those changes live
in a different Pull Request, so this remains 99% ugly for now.

stehefan committed:
Merge pull request #9 from StayWelcome/contact-form

Add unattractive but functional contact-form

Commit:
Merge branch 'better-language-switch' of github.com:StayWelcome/staywelcome.de into better-language-switch
